qmt/batchHarness.py

```diff
@@ -298,4 +307,17 @@ def runBatchPostProc(self, modelFilePath):
         pythonCmd = [pythonName, batchPostProcpath, '\"' + modelFilePath + '\"']
         print('Running {}...'.format(mpiCmd + pythonCmd))
         subprocess.check_call(mpiCmd + pythonCmd,env=my_env)
-        # subprocess.check_call(' '.join(mpiCmd + pythonCmd))
+
+        def purge(dir, pattern):
+            for f in os.listdir(dir):
+                if re.search(pattern, f):
+                    os.remove(os.path.join(dir, f))
+
+        basePath = '/'.join(modelFilePath.split('/')[:-1])
+        solutionsPath = basePath+'/'+\
+                        self.model.modelDict['comsolInfo']['exportDir']
```

Comment:
You're looking for 
```
basePath = os.path.dirname(modelFilePath)
solutionsPath = os.path.join(basePath, self.model.modelDict['comsolInfo']['exportDir'])
```

Reply:
join and surname are fine - but I can't just use the model path. Note that the string manipulation above is to truncate the last directory, since modelFilePath is actually inside one of the geo directories...

Reply:
OK, I just realized what "dirname" does and changed this to your suggestion - thanks!